### DISCUSSION TITLE
After the page is loaded, does single file sends any request to the website when downloading?

### DISCUSSION THREAD

**Discussion by Weebcod3** [gildas-lormeau/SingleFile#938] [General]
or does it just scrape the pages from the cache and stuff that is already on your computer

**Comment by gildas-lormeau** [comment]
It really depends. I know that sometimes the browser can request some resources again without relying on the cache but I do not remember when it happens. It also depends on the browser you're using. SingleFile does however its best to retrieve resources from the cache. You should be able to load a page, disconnect from internet and save the page.
